pages/tx/[hash].tsx

```diff
@@ -566,7 +558,7 @@ const Tx = () => {
       field: t('paymaster_data'),
       content: paymasterData ? (
         <Tooltip title={paymasterData} placement="top">
-          <div className={styles.paymasterData}>{paymasterData}</div>
+          <div className={styles.paymasterData}>{new BigNumber(paymasterData).toFormat()}</div>
```

Comment:
```suggestion
          <div className={styles.paymasterData}>{`0x${paymasterData}`}</div>
```
`paymaster data` is a hex string without `0x`, `new BigNumber(paymasterData)` turns it into a decimal without a radix, so `NaN` will be returned if `paymaster data` has a non-digit char, e.g. `ff`

Instead, I would suggest keeping it in the format of a hex string, the raw format. `hex string` is good for data transformation as it's close to `bytes`